### ISSUE TITLE
freeze the non-temporal parameters 

### ISSUE THREAD

**Issue by nankepan** [hpcaitech/Open-Sora#279]
Hi, I have a question. Should I freeze the non-temporal parameters during training? Thank you!

**Comment by zhengzangw** [comment]
Hello,

We have conducted an experiment on this subject and are happy to share our findings. In summary, we recommend against freezing non-temporal parameters during training.

Initially, we froze the non-temporal parameters and observed that the generated videos were overly static and unsatisfactory. This issue likely stems from the reduced number of trainable parameters, which impairs performance.

Subsequently, we attempted to train all parameters after initially training only the temporal ones. However, the results were inferior compared to training from scratch.

Based on our experiments, we advise against freezing the non-temporal parameters. Attempts to freeze text-related parameters were also unsuccessful. Overall, our recommendation is to train all parameters for optimal results.
